x-pack/solutions/observability/plugins/apm/public/hooks/use_fetcher.tsx

```diff
@@ -82,7 +82,7 @@ type InferResponseType<TReturn> = Exclude<TReturn, undefined> extends Promise<in
   : unknown;
 
 export function useFetcher<TReturn>(
-  fn: (callApmApi: AutoAbortedAPMClient) => TReturn,
+  fn: (callApmApi: AutoAbortedAPMClient, signal: AbortSignal) => TReturn,
```

Comment:
Does this need to receive `signal`? On line 124 the hook creates an instance of the AbortController already

Reply:
The signal only gets applied to the `callApmApi` client, which won't cancel for anything that isn't being called from that. Hence I have to pass the `signal` as a param for anything else wanting to hook into that (like with `apmSourcesAccess` methods).